src/core/variable.jl

```diff
@@ -204,3 +204,28 @@ function variable_tp_oltc_tap(pm::PMs.GenericPowerModel; nw::Int=pm.cnw, bounded
         end
     end
 end
+
+
+"Power drawn by the load at the bus to which it is connected"
+function variable_load_flow(pm::PMs.GenericPowerModel; nw::Int=pm.cnw, cnd::Int=pm.ccnd)
```

Comment:
Can we get by with `variable_load`?  not sure what is meant by `flow` here.  maybe `variable_load_power`?

Reply:
Agreed, let's go with `variable_load`